src/latest_release_note.ts

```diff
@@ -1,28 +1,26 @@
 // TODO: find a better way of loading html into a string
-export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes 🐍📝 (Mar. 10, 2020)</h1>
+export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes 🍰🐍 (Mar. 24, 2020)</h1>
 <p>
-    Thanks for using our extension! We're back with some small updates. <u>Again, please feel free to enable our feature
-        flag in
-        Settings
-        (under the setting titled “<b>deviceSimulatorExpress.previewMode</b>” in the User settings)</u>.
-</p>
+    Thanks for using our extension! <b>Support for micro:bit simulation is now officially released on the
+        Device Simulator Express</b>. 💖✨</p>
 <h2>Changes</h2>
 <p>
-    <h3>Fixes (enabled by default):</h3>
+    <h3>Fixes:</h3>
     <ul>
-        <li>Playing sounds on the CPX simulation now works on MacOS!
-            <ul>
-                <li>Try it out by using <i>cp.play_file(PATH_TO_FILE)</i>.</li>
-            </ul>
-        </li>
-        <li>Previously, the play button on the simulations would sometimes require two clicks. This was fixed.</li>
-        <li>The editor can now support unicode characters (such as Japanese).</li>
+        <li>Fixed serial port selection for serial monitor.</li>
```

Comment:
@vandyliu would you say this is accurate?

Reply:
That was one of the problems. Because that problem was so major, I didn't do much research on if anything else past that was broken so we can write that or just write `Fixed serial monitor not working` or something like that.